docs/guides/dns-over-https.md

```diff
@@ -97,38 +103,116 @@ sudo systemctl start cloudflared
+
+```yaml
+proxy-dns: true
+proxy-dns-port: 5053
+proxy-dns-upstream:
+  - https://1.1.1.1/dns-query
+  - https://1.0.0.1/dns-query
+```
+
+Now install the service via `cloudflared`'s [service command](https://developers.cloudflare.com/argo-tunnel/reference/arguments/#service-command):
+
+```bash
+sudo cloudflared service install
+```
+
+Start the `systemd` service and check its status:
+
+```bash
+sudo systemctl start cloudflared
+sudo systemctl status cloudflared
+```
+
+Now test that it is working! Run the following `dig` command, a response should be returned similar to the one below:
 
+```text
+pi@raspberrypi:~ $ dig @127.0.0.1 -p 5053 google.com
 
-; <<>> DiG 9.10.3-P4-Ubuntu <<>> @127.0.0.1 -p 5053 google.com
+; <<>> DiG 9.11.5-P4-5.1-Raspbian <<>> @127.0.0.1 -p 5053 google.com
 ; (1 server found)
 ;; global options: +cmd
 ;; Got answer:
-;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 65181
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 12157
 ;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1
 
 ;; OPT PSEUDOSECTION:
-; EDNS: version: 0, flags:; udp: 1536
+; EDNS: version: 0, flags:; udp: 4096
+; COOKIE: 22179adb227cd67b (echoed)
 ;; QUESTION SECTION:
 ;google.com.                    IN      A
 
 ;; ANSWER SECTION:
-google.com.             299     IN      A       243.65.127.221
+google.com.             191     IN      A       172.217.22.14
 
-;; Query time: 3 msec
+;; Query time: 0 msec
 ;; SERVER: 127.0.0.1#5053(127.0.0.1)
-;; MSG SIZE  rcvd: 65
+;; WHEN: Wed Dec 04 09:29:50 EET 2019
+;; MSG SIZE  rcvd: 77
 ```
 
 ### Configuring Pi-hole
 
-Finally, configure Pi-hole to use the local `cloudflared` service as the upstream DNS server:
+Finally, configure Pi-hole to use the local `cloudflared` service as the upstream DNS server by specifying `127.0.0.1#5053` as the Custom DNS:
 
 ![Screenshot of Pi-hole configuration](../images/DoHConfig.png)
 
 (don't forget to hit Return or click on `Save`)
 
+### Updating `cloudflared`
+
+If you installed `cloudflared` the automatic way, `cloudflared` will be automatically updated. If you installed it the manual way, you may want to update it from time to time. To do so repeat the steps shown in the beginning after stopping the service and restarting it.
+
+```bash
+# stop the service
+sudo systemctl stop cloudflared
+# fetch and copy the latest version
+wget https://bin.equinox.io/c/VdrWdbjqyF/cloudflared-stable-linux-arm.tgz
+tar -xvzf cloudflared-stable-linux-arm.tgz
+sudo cp ./cloudflared /usr/local/bin
+sudo chmod +x /usr/local/bin/cloudflared
+sudo systemctl start cloudflared
+# verify the service is working fine
+sudo systemctl status cloudflared
+```
+
+### Uninstalling `cloudflared`
+
+#### If installed the manual way
+
+*Courtesy of <https://discourse.pi-hole.net/t/uninstall-cloudflare/21459/3>*
+
+```bash
+sudo systemctl stop cloudflared
+sudo deluser cloudflared
+sudo rm /etc/default/cloudflared
+sudo rm /etc/systemd/system/cloudflared.service
```

Comment:
```suggestion
sudo systemctl stop cloudflared
sudo systemctl disable cloudflared
sudo systemctl daemon-reload
sudo deluser cloudflared
sudo rm /etc/default/cloudflared
sudo rm /etc/systemd/system/cloudflared.service
```

Reply:
Hmm GitHub doesn't allow me to commit the suggestion, I'll do it manually:

>Suggestions cannot be applied on multi-line comments.